common/lib/xmodule/xmodule/tests/test_html_module.py

```diff
@@ -24,6 +24,30 @@ def instantiate_descriptor(**field_data):
     )
 
 
+@ddt.ddt
+class HtmlModuleCourseApiTestCase(unittest.TestCase):
+    """
+    Ensure that student_view_data can handle likely input,
+    and doesn't modify the HTML in any way.
+    This means that it does NOT protect against XSS, escape HTML tags, etc.
+    """
+    @ddt.data(
+        '<h1>Some content</h1>',  # Valid HTML
+        '',
+        None,
+        '<h1>Some content</h',  # Invalid HTML
+        '<script>alert()</script>',  # Does not escape tags
+        '<img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7">',  # Images allowed
+        'short string ' * 100,  # May contain long strings
+    )
+    def test_common_values(self, html):
+        descriptor = Mock()
```

Comment:
@e-kolpakov Decorating the testcase with `@ddt.ddt` doesn't like

```python
class HtmlModuleCourseApiTestCase(unittest.TestCase):
    descripor = Mock()
```
It raises an error where DDT tries to iterate over all class attributes and raises `TypeError: 'Mock' object is not iterable`. Any idea how to avoid this happening?